test/Player-test.js

```diff
@@ -1,2 +1,33 @@
 const { expect } = require('chai');
-const Player = require('../Player.js');
\ No newline at end of file
+const Player = require('../Player.js');
+const Wheel = require('../Wheel.js');
+const data = require('../library.js');
+
+describe('Player', function() {
+
+  it('should have a name', function() {
+    var player = new Player('Ashley');
+    expect(player.name).to.equal('Ashley');
+  });
+
+  it('should have a score', function() {
+    var player = new Player('Ashton', 500);
+    expect(player.score).to.equal(500);
+  });
+
+  it('should have a grand total', function() {
+    var player = new Player('John', 600, 15000)
+    expect(player.grandTotal).to.equal(15000);
+  });
```

Comment:
I'd probably combine the above three tests into an it block that says something like `it should set the correct properties when passed arguments` and just instantiate a single player with all the argument it could possibly take in.